static/docs/command-reference/remote/add.md

```diff
@@ -311,8 +322,8 @@ $ dvc remote add myremote ssh://user@example.com/path/to/dir
 $ dvc remote add myremote hdfs://user@example.com/path/to/dir
 ```
 
-> **Note!** If you are seeing `Unable to load libjvm` error on ubuntu with
-> openjdk-8, try setting JAVA_HOME env variable. This issue is solved in the
+> If you are seeing an `Unable to load libjvm` error on Ubuntu with openjdk-8,
+> try setting the `JAVA_HOME` environment variable. This issue is solved in the
 > [upstream version of pyarrow](https://github.com/apache/arrow/pull/4907) and
 > the fix will be included into the next pyarrow release.
```

Comment:
Is this note still relevant @shcheklein @efiop ? Thanks

Reply:
No longer relevant, indeed. Updated our requirements. We can delete that line. Thank you for the heads up!

Reply:
Thanks. Removing whole note then 🙂